frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -178,6 +178,15 @@ const SampleDetailsContent = ({samplesByID, sampleKindsByID, containersByID, pro
               </Descriptions.Item>
               <Descriptions.Item label="Coordinates">{sample.coordinates || "—"}</Descriptions.Item>
               <Descriptions.Item label="Comment" span={3}>{sample.comment}</Descriptions.Item>
+              { // For flags there's a difference between having a false value (QC failed)
+                // and a null value (QC not performed yet)
+              }
+              <Descriptions.Item label="Quantity Flag">
```

Comment:
The flag are not visible from the list. We should make it visible somehow. I think 2 column for QC flags is too much ... I think we could combine them as QC flag on the list. Maybe we could have a mouse over details popup that give the quality and quantity flags separately.